cowprotocol/cowswap
# Update cookie policy

# Summary

This PR updates and enables Cookie Policy page
https://docs.google.com/document/d/1VHltjyMR2ubWToWqdCzgyIsjl4nEhQEucwQ27HdYxBY/edit#

# To test
- go to `/cookie-policy` page
- make sure the content is correct as defined in the doc
